app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -138,7 +140,9 @@ class SelectionInteractionViewModel private constructor(
         // TODO(#3624): Add warning to user when they exceed the number of allowable selections or are under the minimum
         //  number required.
         selectedItems += itemIndex
+        selectedItemText.set("You may select more choices")
```

Comment:
Could this be logically moved into updateSelectionText to avoid needing to set the item text multiple times here? It may also help with robustness since it ensures that updateSelectionText always uses the most correct text.